engine/consensus/matching/engine.go

```diff
@@ -920,18 +991,18 @@ func (e *Engine) clearPools(sealedIDs []flow.Identifier) error {
 
 // requestPendingReceipts requests the execution receipts of unsealed finalized
 // blocks.
-func (e *Engine) requestPendingReceipts() error {
+func (e *Engine) requestPendingReceipts() (int, error) {
```

Comment:
maybe we could comment what the extra output is, and that it is only useful for debugging purposes